x/pool-incentives/client/cli/cli_test.go

```diff
@@ -30,11 +31,13 @@ func (s *IntegrationTestSuite) SetupSuite() {
 	s.Require().NoError(err)
 }
 
+//TearDownSuite tears down the integration test suite.
 func (s *IntegrationTestSuite) TearDownSuite() {
```

Comment:
We don't really need godocs for these cli test boilerplate things.

It doesn't harm anything to have them, but they're particularly helpful -- we should be in the mindset of getting all the CLI code deleted and rebuilt with proper dev UX. This is work currently going on in the SDK in this regime.